synapse/models/inet.py

```diff
@@ -1801,22 +1807,14 @@ def getModelDefs(self):
                     )),
 
                     ('inet:email:message:attachment', {}, (
-                        ('message', ('inet:email:message', {}), {
-                            'ro': True,
-                            'doc': 'The message containing the attached file.'}),
                         ('file', ('file:bytes', {}), {
-                            'ro': True,
                             'doc': 'The attached file.'}),
                         ('name', ('file:base', {}), {
```

Comment:
NB for consistency - inet:service:message:attachment has :name as file:path vs file:base (file:base makes more sense to me).